src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -68,18 +65,14 @@
   "BlocklistRelease": "Blocklist Release",
   "Book": "Book",
   "BookAvailableButMissing": "Book Available, but Missing",
-  "BookDownloaded": "Book Downloaded",
   "BookEditor": "Book Editor",
-  "BookIndex": "Book Index",
   "BookFileCountBookCountTotalTotalBookCountInterp": "{0} / {1} (Total: {2})",
   "BookFileCounttotalBookCountBooksDownloadedInterp": "{0}/{1} books downloaded",
   "BookFilesCountMessage": "No book files",
-  "BookHasNotAired": "Book has not aired",
+  "BookIndex": "Book Index",
   "BookIsDownloading": "Book is downloading",
   "BookIsDownloadingInterp": "Book is downloading - {0}% {1}",
-  "BookIsNotMonitored": "Book is not monitored",
   "BookList": "Book List",
-  "BookMissingFromDisk": "Book missing from disk",
```

Comment:
Why deleting?

Reply:
There is no existing translation for this.

https://github.com/Readarr/Readarr/blob/d87bf5ae63e5e536560c231612e613404376986d/frontend/src/Activity/History/Details/HistoryDetails.js

will need updating